File: pgm/ABC_Visual_Basic/ABC_402_A.vb
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S As String
        S = Console.ReadLine()
        For Each c As Char in S
            If Char.IsUpper(c) Then
                Console.Write(c)
            End If
        Next
        Console.WriteLine()
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_402_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        string S = Console.ReadLine();
        const string caret = "^";
        const string hyphen = "-";
        const string lbrack = "[";
        const string rbrack = "]";
        const string A = "A";
        const string Z = "Z";
        const string blank = "";
        string pattern = $"{lbrack}{caret}{A}{hyphen}{Z}{rbrack}";
        Console.WriteLine(Regex.Replace(S, pattern, blank));
        Environment.Exit(0);
    }
}